src/v/cluster/archival/tests/ntp_archiver_reupload_test.cc

```diff
@@ -448,11 +442,11 @@ FIXTURE_TEST(
     std::stringstream st;
     stm_manifest.serialize_json(st);
     vlog(test_log.debug, "manifest: {}", st.str());
-    verify_segment_request("500-1-v1.log", stm_manifest);
```

Comment:
This verification ensured a matching local segment, but that isn't the case for this test anymore since the desired segment was merged into the previous segment